rpm/vmu-packer.spec

```
@@ -1,10 +1,10 @@
 Summary: Scripts for using packer for making VMU images
 Name: vmu-packer
-Version: 1.14.2
+Version: 1.15.0
 Release: 1%{?dist}
 License: Apache 2.0
 Source0: %{name}-%{version}.tar.gz
-Requires: packer-io
+Requires: packer
```

```suggestion
%ifarch x86_64
Requires: packer-io
%endif
```
We don't use packer at all on ARM, right? Then we shouldn't need to install it.

`packer-io` is a virtual dependency provided by the `packer-io-bin` build that we have in devops: https://koji.osg-htc.org/koji/packageinfo?packageID=493
we should probably build it into 24-internal for when we get a replacement for osghost3000 (our x86 VM build host).  Then again, maybe it should go into the chtc repos -- @brianhlin , what do you think?